roles/aci/tasks/firewall.yml

```diff
@@ -0,0 +1,22 @@
+---
+- when: r_aci_node_firewall_enabled | bool and not r_aci_node_use_firewalld | bool
+  block:
+  - name: Add iptables allow rules
+    os_firewall_manage_iptables:
+      name: "{{ item.service }}"
+      action: add
+      protocol: "{{ item.port.split('/')[1] }}"
+      port: "{{ item.port.split('/')[0] }}"
+    when: item.cond | default(True)
+    with_items: "{{ r_aci_node_os_firewall_allow }}"
+
+- when: r_aci_node_firewall_enabled | bool and r_aci_node_use_firewalld | bool
+  block:
+  - name: Add firewalld allow rules
+    firewalld:
+      port: "{{ item.port }}"
+      permanent: true
+      immediate: true
+      state: enabled
+    when: item.cond | default(True)
+    with_items: "{{ r_aci_node_os_firewall_all }}"
```

Comment:
Shouldn't this line say `with_items: "{{ r_aci_node_os_firewall_allow }}"` instead of `firewall_all`?

Reply:
From preliminary testing, I confirm this line should indeed state `r_aci_node_os_firewall_allow`. After correcting it locally, the failed step (setting firewalld rules) did indeed pass.